manifests/bootstrap_rpm.pp

```diff
@@ -19,51 +16,48 @@
 
   $katello_rhsm_setup_script = 'katello-rhsm-consumer'
   $katello_rhsm_setup_script_location = "/usr/bin/${katello_rhsm_setup_script}"
+  $candlepin_cert_rpm_alias_filename = 'katello-ca-consumer-latest.noarch.rpm'
 
-  $candlepin_consumer_name = "katello-ca-consumer-${rhsm_hostname}"
-  $candlepin_consumer_summary = "Subscription-manager consumer certificate for Katello instance ${rhsm_hostname}"
-  $candlepin_consumer_description = 'Consumer certificate and post installation script that configures rhsm.'
+  $bootstrap_rpm_name = "katello-ca-consumer-${rhsm_hostname}"
 
   include trusted_ca
   trusted_ca::ca { 'katello_server-host-cert':
     source  => $katello_server_ca_cert,
     require => File[$katello_server_ca_cert],
   }
 
+  package { 'rpm-build':
+    ensure => installed,
+  }
+
   include apache
-  file { $katello_www_pub_dir:
+  file { $rpm_serve_dir:
     ensure => directory,
     owner  => 'apache',
     group  => 'apache',
     mode   => '0755',
```

Comment:
I was wondering why Apache needs to write to this directory. Given it was there before, I don't mind keeping it.